From 861b2528b5a68e17891379eecd88fdea72a6929a Mon Sep 17 00:00:00 2001
From: Robert Schwebel <r.schwebel@pengutronix.de>
Date: Fri, 6 Nov 2009 09:19:58 +0100
Subject: [PATCH 3/5] [fdisk_osf] silence warning

Silence this warning, which crashes when CONFIG_WERROR is active:

cc1: warnings being treated as errors
In file included from util-linux/fdisk.c:622:
util-linux/fdisk_osf.c: In function 'xbsd_writelabel':
util-linux/fdisk_osf.c:961: error: unused parameter 'p'

Signed-off-by: Robert Schwebel <r.schwebel@pengutronix.de>
Signed-off-by: Marc Kleine-Budde <mkl@pengutronix.de>
---
 util-linux/fdisk_osf.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/util-linux/fdisk_osf.c b/util-linux/fdisk_osf.c
index aeccc69..de8df04 100644
--- a/util-linux/fdisk_osf.c
+++ b/util-linux/fdisk_osf.c
@@ -964,6 +964,7 @@ xbsd_writelabel(struct partition *p)
 #if !defined(__alpha__) && !defined(__powerpc__) && !defined(__hppa__)
 	sector = get_start_sect(p) + BSD_LABELSECTOR;
 #else
+	(void)p;	/* silence warning */
 	sector = BSD_LABELSECTOR;
 #endif
 
-- 
1.6.5.2

